frontend/src/pages/RecipientRecord/pages/components/RTTPAHistoryGoalCard.js

```diff
@@ -86,14 +94,16 @@ export default function RTTAPAHistoryGoalCard({ report }) {
               )}
               />
               <GoalDataRow label="Goal" value={goal.goalText} />
-              <GoalDataRow label="Topics" value={goal.goalTopics.join(', ')} />
+              {goal.goalTopics ? (
+                <GoalDataRow label="Topics" value={goal.goalTopics.join(', ')} />
+              ) : null}
             </div>
-            {goal.objectives.length ? (
+            {goal.objectives && goal.objectives.length ? (
```

Comment:
Can these two check for objectives and objectives.length be combined?

Reply:
objectives can be null, so they need to both be checked as optional chaining is not available on the front end